### PR TITLE
Updated readme, livekit-client version

### PR DESCRIPTION
**Pull request by davidzhao**

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# React Native Example App
+
+## Running
+
+```sh
+yarn install
+cd example
+yarn install
```
Comment (davidliu) [inline example/README.md:8]: Just `yarn` (or `yarn bootstrap`) in the root directory should be used to setup the repo/example app.

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* Xcode
+
+Generate the Pod project with:
+
+```sh
+cd ios
+pod install
+```
+
+Then open in Xcode:
```
Comment (davidliu) [inline example/README.md:25]: Did running from command line with `yarn ios` not work?

Comment (davidliu) [reply example/README.md:25]: Oh I guess entitlements may be an issue.

File: package.json [hunk 3]
```diff
@@ -21,31 +21,25 @@
...
-  },
-  "keywords": [
-    "react-native",
-    "ios",
-    "android"
-  ],
   "repository": "https://github.com/livekit/client-sdk-react-native",
   "author": "LiveKit <dl@livekit.io> (https://livekit.io/)",
-  "license": "MIT",
+  "license": "Apache-2.0",
```
Comment (davidliu) [inline package.json:26]: Will need to change the LICENSE file as well.

**Review by davidliu — APPROVED** [review]

File: example/README.md (hunk 2 above)
Comment (davidzhao) [reply example/README.md:25]: yarn ios didn't work for me. also, since I'm on a M1 Mac, iphone simulator didn't work.

**Pull request merged**